src/main/util/consts.py

```diff
@@ -271,3 +271,6 @@ class SPLIT(Enum):
 MAX_DIFF_SYMBOLS = 30
 
 TIMEOUT = 5
+
+TRUE_VALUES_SET = {'true', 't', '1', 'yes', 'y'}
+FALSE_VALUES_SET = {'false', 'f', '0', 'no', 'n'}
```

Comment:
This is also more about cli, maybe we should think about placing it somewhere it src/main/cli? But I'm not sure